src/script/lua_api/l_internal.h

```diff
@@ -30,6 +30,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "common/c_internal.h"
 
 #define luamethod(class, name) {#name, class::l_##name}
+#define luamethod_bc(class, name, alias) {#name, class::l_##name}, {#alias, class::l_##name}
```

Comment:
I'd prefer aliased over alias, as alias implies it's just an alias function whereas alias implies it's a luamethod which is also aliased